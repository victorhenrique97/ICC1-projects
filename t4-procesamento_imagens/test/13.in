2
img1.pgm
conv2.mat

