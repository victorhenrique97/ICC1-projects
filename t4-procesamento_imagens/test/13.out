P2
24 7
15
0 0 0 0 0 0 0 0 0 0 0 0 0 3 3 3 3 0 0 3 3 3 3 0 
0 0 0 0 0 0 0 3 3 3 3 0 3 9 9 9 6 3 3 12 12 12 12 3 
0 0 0 0 0 0 0 3 0 0 0 0 3 9 9 6 3 0 3 12 9 9 12 3 
0 0 0 0 0 0 0 3 3 3 0 0 3 12 9 6 3 0 3 15 12 12 12 3 
0 0 0 0 0 0 0 3 0 0 0 0 3 9 9 6 3 0 3 12 6 3 3 0 
0 0 0 0 0 0 0 3 3 3 3 0 3 9 9 9 6 3 3 9 3 0 0 0 
0 0 0 0 0 0 0 0 0 0 0 0 0 3 3 3 3 0 0 3 0 0 0 0 
